mobile/status.go

```diff
@@ -2429,3 +2440,52 @@ func IntendedPanic(message string) string {
 		panic(err)
 	})
 }
+
+func SwitchToLowMemoryMode() string {
```

Comment:
```suggestion
func ReleaseOSMemory() string {
```

I don't think the `mode` is a proper name, there is no loop or anything managing the "low memory" :thinking:

Reply:
renamed